mozmeao/www-admin
# Dependency security upgrade

This changeset updates Dockerfile to use Python 3.11, so that we can upgrade `mkdocs` to the latest, stable, secure version.

Also, we switch to `pip-tools` as a way to update dependencies, because `hashin` was leaving us with unhashed subdeps that were breaking installation

Resolves #101 
Resolves #98

To test:

* Check out this branch and ensure the `output` dir is empty or non-existent
* `docker-compose build app`
* `docker-compose run app`
* Confirm there is content in the `output` dir

Also check docs build:
* Make a new venv, activate it and and `pip install -r requirements-docs.txt`
* Confirm `mkdocs serve` works
